analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -330,6 +330,23 @@ def construct_config_handler(cls, args, context):
                 if m.group('analyzer') == cls.ANALYZER_NAME:
                     analyzer_config[m.group('key')] = m.group('value')
 
+        # If both --analyzer-config and -config (in --tidyargs) is given then
+        # these need to be merged. Since "HeaderFilterRegex" has a default
+        # value in --analyzer-config, we take --tidyargs stronger so user can
```

Comment:
It would be good to add this information to the `--analyzer-config` / `--tidyargs` options help messages that the options in --tidyargs are stonger.